### PR TITLE
Host form and tenancy request form

### LINKED ISSUE #16: Host listing

### PR DESCRIPTION
**Pull request by ninest**
Closes #16

### CODE REVIEW

**Review by acd124 — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@ yarn-error.log*
 *.tsbuildinfo
 next-env.d.ts
 dev.db
+
+.env
```
Comment (acd124) [inline .gitignore:38]: eof

File: prisma/schema.prisma [hunk 2]
```diff
@@ -27,25 +22,31 @@ model User {
...
 
   requests TenantRequest[]
 
   tenancies Tenancy[]
 }
 
 model MoverUser {
-  id   String @id @default(uuid())
-  user User[]
+  id String @id @default(uuid())
```
Comment (acd124) [inline prisma/schema.prisma:46]: prettier

**Review by acd124 — CHANGES_REQUESTED** [review]
probably some others I missed

**Comment by ninest** [comment]
LGTM

**Pull request merged**